lib/Models/Definition/LoadableStratum.ts

```diff
@@ -24,7 +24,7 @@ export default function LoadableStratum<
         get: function () {
           return undefined;
         },
-        set: function (value: any) {
+        set: function () {
```

Comment:
Same comment for this as the nullFunction below: I suspect removing the parameter will cause problems in some future TypeScript upgrade (and a setter without parameter is strange).

Reply:
Good point, will update